src/components/config_profile/src/profile.cc

```diff
@@ -831,6 +846,33 @@ const std::string& Profile::transport_manager_tcp_adapter_network_interface()
   return transport_manager_tcp_adapter_network_interface_;
 }
 
+#ifdef WEBSOCKET_SERVER_TRANSPORT_SUPPORT
+const std::string& Profile::websocket_server_address() const {
+  return websocket_server_address_;
+}
+
+uint16_t Profile::websocket_server_port() const {
+  return websocket_server_port_;
+}
+#ifdef ENABLE_SECURITY
+const std::string& Profile::ws_server_cert_path() const {
+  return ws_server_cert_path_;
+}
+
+const std::string& Profile::ws_server_key_path() const {
+  return ws_server_key_path_;
+}
+
+const std::string& Profile::ws_server_ca_cert_path() const {
+  return ws_server_ca_cert_path_;
+}
+
+const bool Profile::is_wss_settings_setup() const {
```

Comment:
Can this method name be changed to something like:

`wss_server_supported()`

Reply:
@JackLivio 
renamed in: https://github.com/smartdevicelink/sdl_core/pull/3221/commits/654d6d8d9c2c90574ecc9e4014bc68df2398daa9